recipes/cctbx/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = "2020.8" %}
+
+package:
+  name: cctbx-base
+  version: {{ version }}
+
+source:
+  url: https://github.com/cctbx/cctbx_project/releases/download/v{{ version }}/cctbx-{{ version }}.tar.gz
+  sha256: 38b41bb5e3b968e499a35c30de81147abb946c8de2728e90676f24843348b033
+  patches:
+    - crys3d.patch
+    - cbf.patch  # [win]
+    - libtbx.patch
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - '*/libGL.so.1'  # [linux]
+    - '*/libGLU.so.1'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - boost >=1.65
+    - boost-cpp
+    - eigen
+    - future
+    - hdf5
+    - libtiff
+    - msgpack-c
+    - numpy >=1.14
```

Comment:
This should be empty so the version from conda forge pinning is used instead of the latest.

In staged recipes you might find it doesn't work properly, if so set the versions to match conda-forge-pinning and then delete them once the feedstock is made (it's important to rerender after removing them).

Reply:
Just committed a change.